Makefile

```
@@ -124,7 +124,7 @@ run:
 	fi
 	@mkdir -p logs
 	@echo "$(YELLOW)Starting backend server...$(RESET)"
-	@poetry run nohup uvicorn opendevin.server.listen:app --port $(BACKEND_PORT) > logs/backend_$(shell date +'%Y%m%d_%H%M%S').log 2>&1 &
+	@poetry run uvicorn opendevin.server.listen:app --port $(BACKEND_PORT)
```

Yeah agree, I was almost trying to fix that 😅 just one thing I think we should keep the process execution in the backend because it would interfere with the frontend stdout
```suggestion
	@poetry run uvicorn opendevin.server.listen:app --port $(BACKEND_PORT) &
```

---

Thanks!